From 0e4793502351f698321cae0b24ba1d7f4202f9d2 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 17 Oct 2021 13:29:57 +0200
Subject: [PATCH] generic/config-5.10: UNSET NET_DSA_MSCC_FELIX

Marvell 88E6xxx Ethernet switch fabric support (NET_DSA_MV88E6XXX) [N/m/y/?] n
Ocelot / Felix Ethernet switch support (NET_DSA_MSCC_FELIX) [N/m/y/?] (NEW)
Error in reading or end of file.

make[6]: *** [scripts/kconfig/Makefile:71: syncconfig] Error 1
make[5]: *** [Makefile:603: syncconfig] Error 2
---
 target/linux/generic/config-5.10 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/generic/config-5.10 b/target/linux/generic/config-5.10
index 9809e09b24..2c1d45d5af 100644
--- a/target/linux/generic/config-5.10
+++ b/target/linux/generic/config-5.10
@@ -3833,6 +3833,7 @@ CONFIG_NET_CORE=y
 # CONFIG_NET_DSA_MV88E6171 is not set
 # CONFIG_NET_DSA_MV88E6352 is not set
 # CONFIG_NET_DSA_MV88E6XXX is not set
+# CONFIG_NET_DSA_MSCC_FELIX is not set
 # CONFIG_NET_DSA_MV88E6XXX_NEED_PPU is not set
 # CONFIG_NET_DSA_MV88E6XXX_PTP is not set
 # CONFIG_NET_DSA_QCA8K is not set
-- 
2.30.2

